src/Compilers/Test/Utilities/VisualBasic/BasicTestBase.vb

```diff
@@ -810,8 +799,13 @@ Public MustInherit Class BasicTestBase
 
         Dim fileName = "a.vb"
         Dim syntaxTree = Parse(source, fileName, parseOptions)
-        Dim defaultRefs = If(useLatestFrameworkReferences, LatestVbReferences, DefaultVbReferences)
-        Dim compilation = CreateCompilationWithMscorlib45AndVBRuntime({syntaxTree}, references:=defaultRefs.Append({ValueTupleRef, SystemRuntimeFacadeRef}), options:=If(compilationOptions, TestOptions.ReleaseDll))
+        Dim allReferences As IEnumerable(Of MetadataReference)
+        If useLatestFrameworkReferences Then
```

Comment:
>useLatestFrameworkReferences [](start = 11, length = 28)

Consider moving `useLatestFrameworkReferences` check into `.Add()` call.

Reply:
Will do on the next PR here. 

---
In reply to: [192157327](https://github.com/dotnet/roslyn/pull/25130#discussion_r192157327) [](ancestors = 192157327)